NightscoutServiceKit/Extensions/StoredSettings.swift

```diff
@@ -90,3 +95,39 @@ fileprivate extension Array where Element == RepeatingScheduleValue<Double> {
     }
 
 }
+
+// String conversion methods, adapted from https://stackoverflow.com/questions/40276322/hex-binary-string-conversion-in-swift/40278391#40278391
+fileprivate extension Data {
+    init?(hexadecimalString: String) {
```

Comment:
Would love to see a unit test for this code...I know it is from SO but all the same, it probably would be a trivial test to write.

Reply:
This is copy/paste from four other Loop repos. :)

Reply:
Ah, really...interesting.  Then this is ok, but perhaps we could consider moving it to LoopKit later?

Reply:
There are a ton of tests in the other repos that utilize this function and would break if this didn't work as expected. I think it is safe to leave out a unit test for this one.

Reply:
There are a lot of copy/paste between repos. I agree there should be in a common shared library (perhaps LoopKit), but that's a lot of effort so not sure when it could be done.